traits/api.py

```diff
@@ -230,3 +229,8 @@
         AbstractViewElement.register(ViewElement)
 except ImportError:
     pass
+
+# Backward compatibility for comparison mode constants.
+NO_COMPARE = ComparisonMode.no_compare
```

Comment:
Suggest adding `#:`-style docstrings here. Then we can document these constants in the API documentation (at least to the extent that a search for `OBJECT_IDENTITY_COMPARE` brings up a result).

Reply:
I'll do this in a separate PR.